e2e-tests/tfm/cypress/support/commands.js

```diff
@@ -1,6 +1,9 @@
 // Preserve session cookie
 Cypress.Commands.add('saveSession', require('./utils/saveSession'));
 
+// Assert an element has some exact text
```

Comment:
do we need this comment here?

Reply:
I'm not sure what is best here. Need to declare this command 3x times for different microservices - some command files have comments above each command, others don't.

I have added comment for each command as this seems to be the more prominent approach.

I do think that no command should require a description, it should be self describing.

What do you think (for this PR/command file)?

Reply:
As its already a pattern here, leave it.  But I think the actual function should have a description and not the commands file